rlbench/backend/scene.py

```diff
@@ -556,4 +556,6 @@ def _get_cam_data(cam: VisionSensor, name: str):
         if self._joint_position_action is not None:
             # Store the actual requested joint positions during demo collection
             misc.update({"joint_position_action": self._joint_position_action})
+        joint_poses = [j.get_pose() for j in self.robot.arm.joints]
+        misc.update({'joint_poses': joint_poses})
```

Comment:
This should be in the observation.

Reply:
The use of joint poses is very uncommon. I don't think it's worth adding this to the main observation config